src/containers/Orientation.js

```diff
@@ -1,16 +1,29 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 
+const isMobileDevice = () =>
+  typeof window.orientation !== "undefined" ||
+  navigator.userAgent.indexOf("IEMobile") !== -1;
+
+const isLandscape = () => Math.abs(window.orientation) === 90;
+
+const orientationType = () => {
+  if (isMobileDevice()) {
+    return isLandscape() ? "landscape-primary" : "portrait";
+  } else {
+    return window.screen.orientation.type;
```

Comment:
We should replace `window.screen.orientation` with something else since it's not fully supported yet by all browsers

Reply:
What about this? https://polyfill.io/v2/docs/features/#screen_orientation